js/functions.js

```diff
@@ -0,0 +1,16 @@
+
+// Функция для проверки длины строки
+// eslint-disable-next-line no-unused-vars
+const lengthString = (string, Maxlength) => string.length <= Maxlength;
```

Comment:
В названиях функции стоит юзать глагол. Переменные записываем в camelCase (maxLength)
```suggestion
const checkLengthString = (string, maxLength) => string.length <= maxLength;
```